etcd.py

```diff
@@ -263,6 +266,20 @@ def etcd_arguments(self, data_dir, initial_cluster, cluster_state):
             cluster_state
         ]
 
+        # this section handles etcd version specific flags
+        etcdversion = os.environ.get('ETCDVERSION')
```

Comment:
There is still one corner case left, it breaks upgrade from 3.2 to 3.3 :(
During the upgrade first it starts the old binary in order to get data, than it stops it and starts the new binary.
Current change will try to apply `-listen-metrics-urls` to the old binary and fail if the old version is 3.2

In order to solve it, let's introduce one more argument to this method, `run_old` and pass its value from https://github.com/linki/stups-etcd-cluster/blob/0a914150be28b93ff77b56747b0cf09a010138dc/etcd.py#L474:
```python
    return self.me.etcd_arguments(self.DATA_DIR, peers, cluster_state, self.run_old)
```

And the line 270 has to be changed to:
```python
    etcdversion = os.environ.get('ETCDVERSION_PREV' if run_old else 'ETCDVERSION')
```